app/views/submission_mailer/sentMsg.text.erb

```diff
@@ -0,0 +1,13 @@
+Szanowna Pani/ Szanowny Panie <%= @submission.person.surname %>
+
+Informujemy o zmianie statusu artykułu: "<%= @submission.title %>"
+na "<%= @submission.status %>" 
```

Comment:
No nie do końca - tutaj chodzi o przesłanie decyzji dot. artykułu, czyli bardzo ważnej informacji dla osoby, której tekst jest recenzowany. Proszę to lepiej sformułować.